src/server.c

```diff
@@ -4328,6 +4325,7 @@ sds genRedisInfoString(const char *section) {
             "process_supervised:%s\r\n"
             "run_id:%s\r\n"
             "tcp_port:%i\r\n"
+            "server_time_in_microseconds:%I.%I\r\n"
```

Comment:
Sorry for missing this during the previous rounds, but for consistency:

```suggestion
            "server_time_usec:%I.%I\r\n"
```